core/pom.xml

```diff
@@ -79,7 +79,7 @@
         <dependency>
             <groupId>io.netty</groupId>
             <artifactId>netty-codec</artifactId>
-            <version>${netty.version}</version>
```

Comment:
where is `netty.version` defined?

Reply:
I searched the repo for all instances of text "netty" but couldn't find where it was defined.

https://github.com/search?q=repo%3Aconfluentinc%2Fschema-registry%20netty&type=code

Reply:
can we make the change [here](https://github.com/confluentinc/common/blob/4ae4ac25a06698c16e46ffd36c8218c5785322e1/pom.xml#L87) instead?

Reply:
Opened PR here: https://github.com/confluentinc/common/pull/597